examples/recursion/generate_recursive_proof.sh

```diff
@@ -6,11 +6,15 @@ BACKEND=${BACKEND:-bb}
 # Execute and prove inner circuit (sum)
 mkdir -p ./target/sum
 nargo execute sum_witness --package sum --pedantic-solving
-$BACKEND prove -b ./target/sum.json -w ./target/sum_witness.gz --init_kzg_accumulator --output_format bytes_and_fields -o ./target/sum
-
 # Generate vk for inner circuit
-$BACKEND write_vk -b ./target/sum.json -o ./target/sum --init_kzg_accumulator --output_format fields
+$BACKEND write_vk -b ./target/sum.json -o ./target/sum
+$BACKEND prove -b ./target/sum.json -w ./target/sum_witness.gz -o ./target/sum -k ./target/sum/vk
 
+# Convert binary outputs to JSON format for Noir
+python3 binary_to_fields.py ./target/sum/vk ./target/sum/vk_fields.json
+python3 binary_to_fields.py ./target/sum/vk_hash ./target/sum/vk_hash_fields.json
+python3 binary_to_fields.py ./target/sum/proof ./target/sum/proof_fields.json
+python3 binary_to_fields.py ./target/sum/public_inputs ./target/sum/public_inputs_fields.json
```

Comment:
Can we modify the script to print the result to stdout? we can then pipe directly into Prover.toml